docs/release_notes.md

```diff
@@ -7,6 +7,42 @@ slug: /release_notes
 
 Key features, improvements and bug fixes in the latest releases.
 
+## v0.22.0
+
+Released on November 12, 2025.
+
+### Breaking Changes
+
+:::danger IMPORTANT
+From this release onwards, built-in embedding models will be removed from the released Docker images.
```

Comment:
we ship only the slim edition (without embedding models) Docker image and no longer append the **-slim** suffix to the image tag.